src/Stripe.net/Services/Terminal/Locations/LocationCreateOptions.cs

```diff
@@ -4,5 +4,7 @@ namespace Stripe.Terminal
 
     public class LocationCreateOptions : LocationSharedOptions
     {
+        [JsonProperty("operator_account")]
+        public string OperatorAccount { get; set; }
```

Comment:
`operator_account` can also be used in Location update requests, therefore it should be declared in `LocationSharedOptions` rather than `LocationCreateOptions`.